app/authenticators/custom-jwt.js

```diff
@@ -0,0 +1,66 @@
+import JWT from 'ember-simple-auth-token/authenticators/jwt';
+import { get } from '@ember/object';
+import { assign } from '@ember/polyfills';
+import { Promise } from 'rsvp';
+import { isEmpty } from '@ember/utils';
+import ENV from 'open-event-frontend/config/environment';
+import { inject as service } from '@ember/service';
+
+export default JWT.extend({
+  init() {
+    this._super(...arguments);
+  },
+
+  session: service(),
+
+  handleAuthResponse(response, oldRefreshToken = null) {
+    const token = get(response, this.tokenPropertyName);
+
+    if (isEmpty(token)) {
+      throw new Error('Token is empty. Please check your backend response.');
+    }
+
+    const tokenData = this.getTokenData(token);
+    const expiresAt = get(tokenData, this.tokenExpireName);
+    const tokenExpireData = {};
+
+    tokenExpireData[this.tokenExpireName] = expiresAt;
+
+    if (this.tokenExpirationInvalidateSession) {
+      this.scheduleAccessTokenExpiration(expiresAt);
+    }
+    if (this.refreshAccessTokens) {
+      let refreshToken = null;
+
+      if (oldRefreshToken) {
```

Comment:
this line is not present in ember-simple-auth-token repo code. I have added this to make this able to work for our website.